pkg/tenant/tenants_test.go

```diff
@@ -353,6 +362,23 @@ func (suite *tenantTestSuite) TestTenantDetail() {
 	}
 	_, err = suite.client.TenantDetail(ctx, req.ID)
 	suite.requireError(err, http.StatusNotFound, "tenant not found", "expected error when tenant does not exist")
+
+	// Test VerifyOrg method and pass the resource ID as a value in the database.
```

Comment:
Placing this test after all of the other checks seemed to be the only place where I could get this to work without getting a trtl error.